.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
```

Comment:
We use this label for feature requests.

```suggestion
labels: Priority: Wishlist
```

I wonder if we should also add the "needs design" label here by default?

Reply:
@danrabbit yeah it makes sense that a feature request/wishlist item would require design input.